src/models/migration.js

```diff
@@ -22,18 +22,40 @@ export default async () => {
         phone VARCHAR(15) not null,
         password TEXT not null
         )`
-      ).catch(error => { throw error });
+      ).then(async () => {
+        await query(`INSERT INTO users(
```

Comment:
Why are you programmatically creating this user with this credentials here?

Reply:
Don't do this here. Your tests should create the data themselves.

Reply:
but can I ignore the food menu creation until I create an endpoint for it because there is no endpoint for it?

Reply:
I think you should do that.